local.properties.example

```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
```

Comment:
Should this be checked in given this comment...?

Reply:
Yes, this file should be checked in as it's the `.example` file, it should be renamed without the suffix and is used to set a local versionCode and versionName

Reply:
That comment is a bit misleading then...